contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+challenging. The flat hierarchy of issues can also make navigation and
+categorization tricky. While not all community members might not be comfortable
+using Git directly, I believe it is imperative that as a community we work to
+educate people on a standard set of tools so they can take their experience to
+other projects they may decide to work on in the future.
+
+It is my hope that by increasing the requirements for proposing a change to
+Kubernetes we can prevent important work like generating release notes from
+falling out of the release cycle and in general make working with and speaking
+about Kubernetes an easier and more pleasant experience.
+
+[road to Go 2]: https://blog.golang.org/toward-go2
+
+## Detailed Design
+
+### RFC Template
+
+In order to implement an RFC like process a template of the following form would
+be created
+
+```
+# Title
+## Links
+## Responsible SIG(s)
+## Summary
+## Motivation
+## Examples [optional]
+## Detailed Design
+## Graduation Criteria
+## Drawbacks [optional]
+## Alternatives [optional]
+## Unresolved Questions [optional]
+## Mentors [optional]
+```
+
+where sections marked `[optional]` may be omitted by RFC authors.It is expected
+that most RFCs will begin their lives in discussion with the responsible SIGs,
+preferably in a mailing list so that the discussion is saved for future Kubernauts.
+
+It may be very helpful to provide links to experience reports of other Kubernetes
+users and developers within the `Motivation` section in order to provide additional
+context for the necessity of a proposed change. The `Examples` section could be
+used to provide motivating examples for documentation writers looking to teach
+others how to consume the proposed change once implemented. As an open source
+community we must constantly be looking for ways to better serve newcomers and
+a `Mentors` section could be used to help connect interested parties with elders
+in the community who are able to mentor people towards an implementation.
+
+It is intended for RFCs to be a conversation, the comment part of the acronym,
+so it is not expected for every section of the template to be completed by the
+time an RFC is filed, however, the `Summary`, `Motivation`, and `Examples` section
+should probably be filled out by the time an RFC is filed. Merging an RFC only
+signals agreement of the sections which have been provided thus far and an RFC
+may be completed through a few rounds of review in order to achieve alignment.
+
+### Git and GitHub Implementation
+
+Practically an RFC would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-rfc-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+├── sig-network
+│   └── kube-dns
+├── sig-node
+│   └── kubelet
+├── sig-release
+├── sig-storage
+├── unsorted-to-be-used-by-newcomers-only
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
```

Comment:
Rather than "components", I'd call them subprojects. They might be libraries, features, areas, components, repos, or whatever. We're going to have to formalize that notion in order to unify people organization/leadership and code organization/leadership.